plsql/PlSqlParser.g4

```diff
@@ -520,10 +524,149 @@ alter_index
     : ALTER INDEX old_index_name=index_name RENAME TO new_index_name=index_name ';'
     ;
 
+create_user
+    : CREATE USER
+      user_object_name
+        ( identified_by
+          | identified_other_clause
+          | user_tablespace_clause
+          | quota_clause
+          | profile_clause
+          | password_expire_clause
+          | user_lock_clause
+          | user_editions_clause
+          | container_clause
+        )+ ';'
+    ;
+
+// The standard clauses only permit one user per statement.
+// The proxy clause allows multiple users for a proxy designation.
+alter_user
+    : ALTER USER
+      ( ( user_object_name
+          ( alter_identified_by
+          | identified_other_clause
+          | user_tablespace_clause
+          | quota_clause
+          | profile_clause
+          | user_default_role_clause
+          | password_expire_clause
+          | user_lock_clause
+          | alter_user_editions_clause
+          | container_clause
+          | container_data_clause
+          )+
+        )
+      | ( user_object_name (',' user_object_name)*
+          proxy_clause
+        )
+      ) ';'
+    ;
```

Comment:
Contains excess parentheses. They make worse readability. Try to get rid of them.